capi/VideoDecoderCapi.cpp

```diff
@@ -18,8 +18,8 @@
 #include "config.h"
 #endif
 #include "VideoDecoderCapi.h"
-#include "interface/VideoDecoderInterface.h"
-#include "interface/VideoDecoderHost.h"
+#include "VideoDecoderInterface.h"
+#include "VideoDecoderHost.h"
```

Comment:
I would prefer to merge only the external header error with this patch and then create other for the internal includes. Can this be taken care of?